source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -52,6 +52,14 @@ private string DetectUser()
         }
 
         [AcceptVerbs(HttpVerbs.Get)]
+        [CommonJobsAuthorize(Roles = "EmployeeManagers")]
+        public ActionResult ReportDashboard()
+        {
+            //TODO: Needs UI. Will call the API for the list (GetEvaluatedEmployees)
```

Comment:
Actually that call is async and depends on the link (Ver evaluaciones). That's for when the EmployeeManager sees a finished evaluation and wants to see the auto & company califications.